tests/wasmbus.rs

```diff
@@ -269,10 +274,10 @@ async fn wasmbus() -> anyhow::Result<()> {
             [],
         ) => {
             // TODO: Validate `issuer`
-            ensure!(cluster_issuers == Some("TODO".into()));
+            ensure!(cluster_issuers == Some(cluster_key.public_key()));
```

Comment:
Could you file an issue to test that hosts can accept invocations from another issuer?

Reply:
Added more information into #483, we still have to implement the actual invocation verification and it should be done

Reply:
I think what I'm describing is a little different. #483 captures that the host needs to verify invocations. I was proposing we add the happy path to our tests, ie start two hosts with different cluster seeds but with shared issuers and confirm that invocations work across hosts

The motivation for this comes from doing some manual e2e testing during the weekend: I realized setting cluster issuers wasn't implemented yet when I ran into errors from a _provider_ refusing to process an invocation, because it hadn't been given a list of valid issuers by the host

Reply:
Implemented in https://github.com/wasmCloud/wasmCloud/pull/496